server/json-logger.h

```diff
@@ -17,7 +17,7 @@ class JsonLogger : vk::not_copyable {
 public:
   friend class vk::singleton<JsonLogger>;
 
-  void init(int64_t release_version) noexcept;
+  void init(int64_t release_version, vk::string_view hostname) noexcept;
```

Comment:
Is it okay, that args don't have `const` qualifier, but in implementation they have?